### DISCUSSION TITLE
part-ii/2021-120/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#401] [Question comments]
Part II 2021 Principles of Statistics: Paper 4, Section II, J 

https://questions.tripos.org/part-ii/2021-120/

**Comment by ArthurConmy** [comment]
I'm confident-ish about 

<details>
<summary>a)</summary>

Quadratic risk minimisers are the expectations of $\theta$ in the posterior:

$\theta_{BAYES} = \frac{\alpha + x}{ 1+\beta }$

quadratic risk is bias squared plus variance:

$\frac{(\alpha - \beta\theta)^2}{(1+\beta)^2} + \frac{\theta}{(1+\beta)^2}$

which always has non-zero $\theta^2$ coefficient and hence is never minimax. 
</details> 

Less confident, but am more curious about

<details>
<summary>b)</summary>

$$ \mu_{BAYES} = \left( \frac{\beta + 1}{\beta + 2} \right)^{\alpha + x -1} $$

which we can then compute the risk of, since we can isolate $\mathbb{ E}[t^{X}]$ expressions, and $X$ is Poisson and hence has given PGF. I get a horrible risk with different $e^{-k \theta}$ powers that can't be made constant in $\theta$.

</details>
